coordinator/src/main/java/org/apache/uniffle/coordinator/SimpleClusterManager.java

```diff
@@ -319,6 +345,20 @@ public boolean deleteLostServerById(String serverId) {
     return false;
   }
 
+  @Override
+  public boolean addExcludeNodes(List<String> excludeNodeIds) {
+    boolean addFlag;
+    try {
+      putInExcludeNodesFile(excludeNodeIds);
+      excludeNodes.addAll(excludeNodeIds);
+      addFlag = true;
+    } catch (IOException e) {
+      addFlag = false;
+      LOG.warn("Because {}, failed to add blacklist.", e.getMessage());
+    }
+    return addFlag;
```

Comment:
```suggestion
    try {
      putInExcludeNodesFile(excludeNodeIds);
      excludeNodes.addAll(excludeNodeIds);
      return true;
    } catch (IOException e) {
      LOG.warn("Because {}, failed to add blacklist.", e.getMessage());
      return false;
    }
```